service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/StandardDirectionsOrderIssuedEventHandlerTest.java

```diff
@@ -76,7 +72,7 @@
     private StandardDirectionsOrderIssuedEventHandler standardDirectionsOrderIssuedEventHandler;
 
     @Test
-    void shouldNotifyCafcassOfIssuedSDOWhenNoticeOfProceedingsIsDisabled() {
+    void shouldNotifyCafcassOfIssuedSDOAndNoticeOfProceedingsWhenSendNoticeOfProceedings() {
```

Comment:
`shouldNotifyCafcassOfIssuedSDOAndNoticeOfProceedingsWhenSendNoticeOfProceedings`

we are always sending notice of proceedings so could just be 

`shouldNotifyCafcassOfIssuedSDOAndNoticeOfProceedings` or
`shouldNotifyCafcassOfIssuedSDOAndNoticeOfProceedingsWhenIssuingAnSDO`